templates/resetPasswordEmail.tmpl

```diff
@@ -0,0 +1,223 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
```

Comment:
I didn't write it, mjml was used from a super simple initial email template and it wrote all of the rest of it including that

Reply:
But someone will have to edit it sooner or later, and I wouldn't blame someone for sticking a fondue fork into their eyeballs after trying to edit it. This might be for a later PR, but we really should look into ways of doing this that don't involve writing all that out by hand.

Reply:
I can add the mjml files for the easy editing of emails? They are 25 lines each

Reply:
That may be worthwhile, yeah - and you can also add a `//go:generate <command to generate them>` line so that future users can just run `go generate ./templates` to re-build the HTML from mjml.

Reply:
Tomorrow problem

Reply:
I've done this now